torch/utils/_python_dispatch.py

```diff
@@ -142,13 +142,23 @@ def is_traceable_wrapper_subclass(t):
     is 'traceable' with torch.compile.
     In order for a tensor subclass to support TorchDispatchMode-style tracing in PT2,
     It must implement two magic methods: __tensor_flatten__ and __tensor_unflatten__.
-    It is also expected to obey some restrictions around traceability and aliasing
-    (TODO: add clear documentation around this.)
+    It is also expected to obey some restrictions around traceability and aliasing.
+
+    Expected signatures:
+        attrs, ctx = t.__tensor_flatten__()
+            attrs: list of attribute name strings for inner tensors
+            ctx: dict containing any other subclass-specific metadata needed for unflattening
+
+        t = MySubClass.__tensor_unflatten__(inner_tensors, ctx, outer_size)
+            inner_tensors: dict mapping attribute name -> tensor for each inner tensor
+            ctx: dict with subclass metadata in the form that __tensor_flatten__() produces
+            outer_size: expected (possibly symbolic) size that the returned subclass
```

Comment:
sorry one last dumb question, I actually dont know from this doc what I need to update on my subclass. __unflatten__ will consume these, objects, should this just be dropped: https://github.com/pytorch-labs/float8_experimental/blob/77386ba70d5a298ed712ec3a213b1e07ceec18a9/float8_experimental/float8_tensor.py#L125
?